.github/workflows/release.yml

```diff
@@ -36,10 +36,10 @@ jobs:
       id: destination
       run: |
         INPUT_DESTINATION=${{ github.event.inputs.destination }}
-        echo "destination=${INPUT_DESTINATION:-"appstore"}" >> $GITHUB_OUTPUT
+        echo "destination=${INPUT_DESTINATION:-"testflight"}" >> $GITHUB_OUTPUT
 
     - name: Assert release branch
-      if: steps.destination.outputs.destination == 'appstore'
+      if: steps.destination.outputs.destination == 'appstore' || steps.destination.outputs.destination == 'testflight'
```

Comment:
`appstore` and `testflight` are the only options on iOS, so we can remove the `if:` statement altogether 👍